src/app/public/src/modules/swagger/swagger.component.ts

```diff
@@ -0,0 +1,33 @@
+import { AfterViewInit, Component, Input, OnInit } from '@angular/core';
+const swagger = require ('swagger-ui-dist');
+
+@Component({
+  selector: 'stache-swagger',
+  templateUrl: './swagger.component.html'
+})
+export class StacheSwaggerComponent implements OnInit, AfterViewInit {
+  @Input()
+  public swaggerUrl: string;
+
+  private swaggerConfig = {
```

Comment:
Is this something that can live in the skyuxConfig similar to the analytics stuff?
https://github.com/blackbaud/stache2/blob/master/skyuxconfig.json#L13

Reply:
I wasn't sure if this was something we wanted to override, but yeah, I can move it to the config. I was initially going to put it there anyway.

Reply:
And we can't pull all of it out regardless. Since some of that stuff isn't static.

Reply:
Yeah, it's not worth pulling this out. We would literally only have the default URL and the template in here.